cli/lsp/tsc.rs

```diff
@@ -2669,13 +2531,22 @@ fn op_is_cancelled(state: &mut OpState) -> Result<bool, AnyError> {
 fn op_load(
   state: &mut OpState,
   args: SpecifierArgs,
-) -> Result<Option<String>, AnyError> {
+) -> Result<Value, AnyError> {
```

Comment:
This `op_load` was previously never called. That's why its return type not being correct didn't matter.